cmd/descheduler/app/server.go

```diff
@@ -62,13 +63,19 @@ func NewDeschedulerCommand(out io.Writer) *cobra.Command {
 				return
 			}
 			var factory registry.LogFormatFactory
+
+			switch s.Logging.Format {
+			case "json":
```

Comment:
@wakeful are there any other cases to consider? Based on https://kubernetes.io/blog/2020/09/04/kubernetes-1-19-introducing-structured-logs/ there's currently only json. Also from
```
      --logging-format string                    Sets the log format. Permitted formats: "text", "json". Non-default formats don't honor these flags: --add-dir-header, --alsologtostderr, --log-backtrace-at, --log_dir, --log_file, --log_file_max_size, --logtostderr, --skip-headers, --skip-log-headers, --stderrthreshold, --log-flush-frequency.\nNon-default choices are currently alpha and subject to change without warning. (default "text")
```
Only `json` and `text` values are valid. Using simple `if s.Logging.Format == "json"` is more preferable atm.

Reply:
done